.gitignore

```diff
@@ -6,24 +6,39 @@ target
 *~
 .project
 .classpath
-.idea/
-*.iml
 release.properties
 **/.DS_Store
-*.class
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
-target
```

Comment:
is the target directory generated by maven? Could we want to exclude it for some reason?

Reply:
It was duplicated, so this is just deduplicating it. Target is still ignored :-)